ts/output/chtml/FontData.ts

```diff
@@ -209,6 +213,35 @@ export class CHTMLFontData extends FontData<CHTMLCharOptions, CHTMLVariantData,
     return styles;
   }
 
+  /**
+   * @param {StyleList} styles  The style list to add characters to
+   */
+  protected allStyles(styles: StyleList) {
+    //
+    //  Create styles needed for the delimiters
+    //
+    for (const n of Object.keys(this.delimiters)) {
+      const N = parseInt(n);
+      this.addDelimiterStyles(styles, N, this.delimiters[N]);
+    }
+    //
+    //  Add style for all character data
+    //
+    for (const name of Object.keys(this.variant)) {
+      const variant = this.variant[name];
+      const vletter = variant.letter;
+      for (const n of Object.keys(variant.chars)) {
+        const N = parseInt(n);
```

Comment:
Why do you need to parse that here? `variant.chars[n]` should return the value and in case the `continue` fires first, you don't need the actual `number`.

Reply:
This is just to make Typescript happy.  Since `Object.keys()` returns `string[]`, while `variant.char` is indexed by numbers; it is a `CharMap`:

https://github.com/mathjax/MathJax-src/blob/2dd53ce6c8af3c9cceba0baf014ea9b065130774/ts/output/common/FontData.ts#L58-L60

I could change that to being string indexed, if you prefer.  That doesn't seem to cause any type errors.

I've had to do this a couple of other times, as there are several other namer-indexed objects.  I could try converting them all to strings and see if anything blows up.

Reply:
I see. Strictly speaking Typescript is of course correct, as Object keys are always strings, so I suppose it is safer this way.

Reply:
OK, we can see about perhaps changing over to string-indexed objects rather than number-indexed in a future code clean-up